pkg/operator/docker/resourcestatus/status.go

```diff
@@ -0,0 +1,43 @@
+package resourcestatus
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/client"
+
+	"github.com/seal-io/walrus/pkg/dao/types/status"
+)
+
+// resourceTypes indicate supported resource type and function to get status.
+var resourceTypes map[string]getStatusFunc
+
+// getStatusFunc is function use resource id to get resource status.
+type getStatusFunc func(ctx context.Context, client *client.Client, resourceType, name string) (*status.Status, error)
+
+func init() {
+	resourceTypes = map[string]getStatusFunc{
+		"docker_container": getContainerStatus,
```

Comment:
Do we need to include other resource types like docker_image?

Reply:
We can make further enhancements based on requirements or what we provide in our built-in templates.